_security/access-control/read-only-role.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Read only roles
+parent: Access control
+nav_order: 150
+---
+
+# Read only roles
+
+As with any roles in OpenSearch, read only roles can be configured using three methods: `yml` configuration files, API and OpenSearch Dashboards. The most user friendly approach for anyone getting familiar with Roles and role mappings is to use OpenSearch Dashboards, as it is easy to navigate the creation of roles and assignment to users. The basic steps of creating roles/mappings/users are outlines here (https://opensearch.org/docs/latest/security/access-control/users-roles/)
+
+## Basic read only role
+
+If you are looking to create a role to access OpenSearch Dashboards, view existing dashboards/visualizations and query different indexes, assuming you want the user to have read access to all indexes and tenants, you can add cluster permission group of `cluster_composite_ops_ro`, `read` access to all indexes (`*`) and `read` access to all tenants
```

Comment:
I would consider explaining each of these. Without knowing the background it is not super obvious what `cluster_composite_ops_ro` is for. Likewise it may not be obvious why you restate the permissions at different levels so you may want to clarify this.

Reply:
@scrawfor99 good point, I've expanded on this, let me know if below is better:
## Basic read only role

If you are looking to create a role to access OpenSearch Dashboards, view existing dashboards/visualizations and query different indexes, assuming you want the user to have read access to all indexes and tenants, you can use the following permissions.
### Cluster permission
Cluster permission is needed to access cluster wide resources like visualizations and dashboards. For read only user you add already created `cluster_composite_ops_ro` permission.
### Indexes permission
In order to view visualizations, user needs to have access to the index that was used to create it, In this can you can add `read` permission to all (`*`) indexes. 
### Tenant permissions
If you are using tenants to separate work area between different teams/projects, visualizations and dashboards are saved in specific tenant. To be able to view these the user needs `read` access to the correct tenant. To keep things more straightforward we are giving user access to all (`*`) tenants.